File: Node.js/calculateIncome.js
module.exports = function (income, ari, year) {
	
	//for 1st year, returns the income entered by the user
	if(year == 1 || ari == 0){
		return income;
	}
	
	// else, calculates income and returns
	else{
		
		income = income + ((income*ari)/100);
		
		//ceiling income to 2 decimal points
		income = Math.ceil(income*100)/100; 
		return income;
	}
	
};

File: IN JAVA/CalculateIncome.java
package com.retireup;

public class CalculateIncome {

	//static private int count = 0;
	
	public static float calculateIncome(float income, float ari, int year){
		
		//System.out.println("Income and Taxes for 50 years");
		if(year == 1){
			return income;
		}
		
		else{
		income = income + ((income*ari)/100);
		income = Math.round(income*100.0f)/100.0f;		
				
		return income;
		}
	}
}
